apps/meteor/ee/client/omnichannel/cannedResponses/components/cannedResponseForm.tsx

```diff
@@ -3,7 +3,7 @@ import { Box, Field, TextInput } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import React, { FC } from 'react';
 
-import AutoCompleteDepartment from '../../../../../client/components/AutoCompleteDepartment';
+import { AutoCompleteDepartment } from '../../../../../client/components/AutoCompleteDepartment';
```

Comment:
Have you checked if the folder `client/components` have an index file? If so, it makes sense importing from there, if not, maybe use this opportunity to create one?

Reply:
Well, I have spoken with Tasso about it, and he told me this folder deserves a future attention on changing files to specific modules and other places... so, doing a barrel here will not be helpful for this future task at all. What I mean is, stay as it is for now...